README.md

```diff
@@ -19,183 +19,245 @@ The following is the approximate sequence of the topics during the program but i
 ### Week 0 - Pre-Work
 Before beginning the program, apprentices meet weekly for 4 weeks to learn JavaScript fundamentals.
 
-### Week 1 - Onboarding & Developer Tools
+### Week 1 - Introduction & Developer Tools
```

Comment:
Why did you change this?

Reply:
It's mentioned on the Roadmap Doc as Introduction & Developer Tools rather than onboarding, so I changed it to make it consistent with the RoadMap Doc itself. Should I change it back?